rmm/datto/Zorus Filtering Install  Uninstall Windows/command.bat

```diff
@@ -0,0 +1,159 @@
+function script:Get-WebClientWithProxy() {

```

Comment:
I think it should, but when you export the component file from Datto's portal, even if the file is set to PowerShell it will output it as a bat. I'd rather not find out if that breaks compatibility 🤣.